aspnetcore/client-side/dotnet-interop.md

```diff
@@ -427,6 +427,18 @@ dotnet serve -d:bin/$(Configuration)/{TARGET FRAMEWORK}/publish
 
 In the preceding example, the `{TARGET FRAMEWORK}` placeholder is the [target framework moniker](/dotnet/standard/frameworks).
 
+:::moniker range=">= aspnetcore-10.0"
+
+To enable [Hot Reload](xref:test/hot-reload) for WebAssembly, set the `WasmEnableHotReload` MSBuild property to `true` in the app's project file:
```

Comment:
I have probably stated the issue incorrectly - sorry for that.

The property is set by default when building in `Debug` configuration. It's not set (=false) when building in any other configuration.

If users want to use custom configuration name when debugging (like `DebugWebAssembly` etc) and want to enable HotReload, they need to set the property to true. And similarly, if they want to disable HotReload for configuration `Debug` they need to set the property to false.

Reply:
This applies for both Blazor WebAssembly and .NET on WebAssembly (without Blazor). I'm not sure that this is the right document